src/index.js

```diff
@@ -2,3 +2,21 @@ import chordParserFactory from './parser/chordParserFactory';
 import chordRendererFactory from './renderer/chordRendererFactory';
 
 export { chordParserFactory, chordRendererFactory };
+
+
+/**
+ * @typedef { import('./parser/chordParserFactory').Chord } Chord
+ * @export
+ */
+/**
+ * @typedef { import('./parser/chordParserFactory').ChordInput } ChordInput
+ * @export
+ */
+/**
+ * @typedef { import('./parser/chordParserFactory').FormattedChord } FormattedChord
+ * @export
+ */
+/**
+ * @typedef { import('./parser/chordParserFactory').NormalizedChord } NormalizedChord
+ * @export
+ */
```

Comment:
I'm puzzled. Why is this necessary if all the files in the repo get their corresponding d.ts file? Maybe just adding @export to the original declaration would do the trick?

Reply:
Unfortunately, this is what I then get:
![image](https://user-images.githubusercontent.com/35939574/96276621-3e0b2080-0fa1-11eb-81d5-1fb748416347.png)

Reply:
But `Chord` should not be exported from the main entry point.

However, does your editor recognize `parsed` of being of type `Chord` ?

```javascript
const parseChord = chordParserFactory();
const parsed = parseChord('Cm7');
```

I tried on my own and it seems to work out of the box with the jsdoc annotations only